### PR TITLE
Mac Silicon

### PR DESCRIPTION
**Pull request by notFloran**
Add supports of Mac Silicon.

Include some cleanup like drush launcher or the simplification of the openssl certificates part

### CODE REVIEW

File: roles/dotfiles/templates/zshrc.j2 [hunk 1]
```diff
@@ -14,7 +14,14 @@ source $ZSH/oh-my-zsh.sh
 
 DEFAULT_USER="{{ lookup('env','USER') }}"
 
-export PATH=$HOME/bin:/usr/local/bin:/usr/local/sbin:$PATH
+export PATH=$HOME/bin:$PATH
+
+if type brew &>/dev/null; then
```
Comment (notFloran) [inline roles/dotfiles/templates/zshrc.j2:19]: It's documented likes that, see `brew info zsh-completions`

File: roles/drush/tasks/main.yml [hunk 2]
```diff
@@ -1,21 +1,5 @@
 ---
 
-- name: Check if drush launcher is installed.
```
Comment (notFloran) [inline roles/drush/tasks/main.yml:3]: drush launcher is archived, not needed anymore

File: roles/mkcert/tasks/main.yml [hunk 3]
```diff
@@ -25,5 +25,6 @@
 
 - name: Replace certifi CA 
   shell: cp {{ openssl_ca_file }} $(python -m certifi)
+  ignore_errors: true
```
Comment (notFloran) [inline roles/mkcert/tasks/main.yml:28]: I have error that certifi CA and openssl ca are the same and it throw error

File: scripts/bootstrap.sh [hunk 4]
```diff
@@ -8,8 +8,8 @@ xcode-select --install
 
 echo "Checking for Homebrew..."
 if test ! $(which brew); then
-  echo "Installing homebrew..."
-  /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+  echo "You must install homebrew, see https://brew.sh/"
```
Comment (notFloran) [inline scripts/bootstrap.sh:11]: Now after brew install user must do : 

```
echo 'eval $(/opt/homebrew/bin/brew shellenv)' >> /Users/"YOUR USER NAME"/.zprofile
eval $(/opt/homebrew/bin/brew shellenv)
```

I didn't want to waste time automating this so I took the homebrew installation out of the script.

**Review by nclsHart — APPROVED** [review]

**Pull request merged**